lib/src/main/java/uk/gov/di/ipv/cri/experian/kbv/api/service/KeyStoreService.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.di.ipv.cri.experian.kbv.api.service;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import software.amazon.lambda.powertools.parameters.SecretsProvider;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Base64;
+
+public class KeyStoreService {
+    private final SecretsProvider secretsProvider;
+    public static final String KBV_API_KEYSTORE = "/dev/di-ipv-cri-experian-kbv-api/keystore";
+    public static final String KBV_API_KEYSTORE_PASSWORD = "/dev/di-ipv-cri-experian-kbv-api/keystore-password";
+    private static final Logger LOGGER = LoggerFactory.getLogger(KeyStoreService.class);
```

Comment:
I think the log4 dependencies will need to be re-instated for logging to work. It's worth checking